app/errors/authentication/errors.rb

```diff
@@ -0,0 +1,27 @@
+module Authentication
+  module Errors
```

Comment:
I decided to move this module from `app/services/authentication/errors.rb`. Given that we have a dedicated `app/errors` this seems like a more sensible place, but I can move it back if people would prefer that.